lib/stripe/stripe_client.rb

```diff
@@ -120,36 +120,38 @@ def execute_request(method, path,
       params = Util.objects_to_ids(params)
       url = api_url(path, api_base)
 
+      body = nil
+      query_string = nil
+
       case method.to_s.downcase.to_sym
       when :get, :head, :delete
-        # Make params into GET parameters
-        url += "#{URI.parse(url).query ? '&' : '?'}#{Util.encode_parameters(params)}" if params && params.any?
-        payload = nil
+        query_string = Util.encode_parameters(params) if params && params.any?
+        url += "#{URI.parse(url).query ? '&' : '?'}#{query_string}" unless query_string.nil?
       else
-        payload = if headers[:content_type] && headers[:content_type] == "multipart/form-data"
-                    params
-                  else
-                    Util.encode_parameters(params)
-                  end
+        body = if headers[:content_type] && headers[:content_type] == "multipart/form-data"
+                 params
+               else
+                 Util.encode_parameters(params)
+               end
       end
 
       headers = request_headers(api_key, method)
                 .update(Util.normalize_headers(headers))
 
       # stores information on the request we're about to make so that we don't
       # have to pass as many parameters around for logging.
-      context = RequestLogContext.new(
-        account: headers["Stripe-Account"],
-        api_key: api_key,
-        api_version: headers["Stripe-Version"],
-        idempotency_key: headers["Idempotency-Key"],
-        method: method,
-        path: path,
-        payload: payload
-      )
+      context = RequestLogContext.new
+      context.account         = headers["Stripe-Account"]
+      context.api_key         = api_key
+      context.api_version     = headers["Stripe-Version"]
+      context.body            = body
+      context.idempotency_key = headers["Idempotency-Key"]
+      context.method          = method
+      context.path            = path
+      context.query_string    = query_string
```

Comment:
I ended up changing this to just use the accessors because the Linter screamed when I added another parameter to `RequestLogContext`'s constructor. It's just as name-safe and readable, so not too big of a deal.